NCS.DSS.EmploymentProgression/GetEmploymentProgression/Function/EmploymentProgressionGetTrigger.cs

```diff
@@ -54,44 +50,50 @@ IGuidHelper guidHelper
         public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = RouteValue)]
             HttpRequest req, string customerId)
         {
-            _loggerHelper.LogInformation("Started EmploymentProgressionGetTrigger");
+            var functionName = nameof(EmploymentProgressionGetTrigger);
 
-            var correlationId = _httpRequestHelper.GetDssCorrelationId(req);
-            var correlationGuid = _guidHelper.ValidateAndGetGuid(correlationId);
+            _logger.LogInformation("Function {FunctionName} has been invoked", functionName);
 
+            var correlationId = _httpRequestHelper.GetDssCorrelationId(req);
             var touchpointId = _httpRequestHelper.GetDssTouchpointId(req);
             if (string.IsNullOrEmpty(touchpointId))
             {
-                _loggerHelper.LogInformation($"[{correlationGuid}] Unable to locate 'TouchpointId' in request header.");
+                _logger.LogWarning("{CorrelationId} Unable to locate 'TouchpointId' in request header.", correlationId);
                 return new BadRequestResult();
             }
 
             var ApimURL = _httpRequestHelper.GetDssApimUrl(req);
             if (string.IsNullOrEmpty(ApimURL))
             {
-                _loggerHelper.LogInformation($"[{correlationGuid}] Unable to locate 'apimurl' in request header");
+                _logger.LogWarning("{CorrelationId} Unable to locate 'apimurl' in request header", correlationId);
                 return new BadRequestResult();
             }
 
             if (!Guid.TryParse(customerId, out var customerGuid))
             {
-                _loggerHelper.LogInformation($"[{correlationGuid}] Unable to parse 'customerId' to a Guid: {customerId}");
+                _logger.LogWarning("{CorrelationId} Unable to parse 'customerId' to a Guid: {customerId}", correlationId, customerId);
                 return new BadRequestObjectResult(customerId);
             }
 
-            if (!await _resourceHelper.DoesCustomerExist(customerGuid))
+            if (!await _cosmosDbProvider.DoesCustomerResourceExist(customerGuid))
```

Comment:
Also, this is calling a method so can we please add a "attempting to" log and a "Success" log rather than just a failure log